qa/L0_java_simple_example/test.sh

```diff
@@ -0,0 +1,170 @@
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# Set up test files based on installation instructions
+# https://github.com/bytedeco/javacpp-presets/blob/master/tritonserver/README.md
+rm -r javacpp-presets
+git clone https://github.com/bytedeco/javacpp-presets.git
+cd javacpp-presets
+mvn clean install --projects .,tritonserver,cuda,tensorrt
+mvn clean install -f platform --projects ../tritonserver/platform -Djavacpp.platform.host
+mvn clean install -f platform --projects ../cuda/platform -Djavacpp.platform.host
+mvn clean install -f platform --projects ../tensorrt/platform -Djavacpp.platform.host
+cd ..
+
+CLIENT_LOG="client_cpu_only.log"
+MODEL_REPO=`pwd`/models
+SAMPLES_REPO=`pwd`/javacpp-presets/tritonserver/samples
+BASE_COMMAND="mvn clean compile -f $SAMPLES_REPO exec:java -Djavacpp.platform=linux-x86_64"
+source ../common/util.sh
+
+# Create local model repository
+rm -r models
+cp -r `pwd`/../L0_simple_ensemble/models .
+mkdir ${MODEL_REPO}/ensemble_add_sub_int32_int32_int32/1
+
+rm -f *.log
+RET=0
+
+function run_cpu_tests() {
+    # Run with default settings
+    $BASE_COMMAND -Dexec.args="-r $MODEL_REPO" >>$CLIENT_LOG 2>&1
+    if [ $? -ne 0 ]; then
+        echo -e "Failed to run: ${BASE_COMMAND} -Dexec.args=\"-r ${MODEL_REPO}\""
+        RET=1
+    fi
+
+    if [ `grep -c "1 - 1 = 0" ${CLIENT_LOG}` != "18" ]; then
+        echo -e "\n***\n*** Failed. Expected 18 '1 - 1 = 0'\n***"
+        RET=1
+    fi
+
+    # Run with verbose logging
+    $BASE_COMMAND -Dexec.args="-r $MODEL_REPO -v" >>$CLIENT_LOG 2>&1
+    if [ $? -ne 0 ]; then
+        echo -e "Failed to run: ${BASE_COMMAND} -Dexec.args=\"-r ${MODEL_REPO} -v\""
+        RET=1
+    fi
+
+    if [ `grep -c "Server side auto-completed config" ${CLIENT_LOG}` != "2" ]; then
+        echo -e "\n***\n*** Failed. Expected 'Server side auto-completed config'\n***"
+        RET=1
+    fi
+
+    # Run with memory set to system
+    $BASE_COMMAND -Dexec.args="-r $MODEL_REPO -m system" >>$CLIENT_LOG 2>&1
+    if [ $? -ne 0 ]; then
+        echo -e "Failed to run: ${BASE_COMMAND} -Dexec.args=\"-r ${MODEL_REPO} -m system\""
+        RET=1
+    fi
+
+    if [ `grep -c "OUTPUT0 is stored in system memory" ${CLIENT_LOG}` != "9" ]; then
+        echo -e "\n***\n*** Failed. Expected 9 'OUTPUT0 is stored in system memory'\n***"
+        RET=1
+    fi
+
+    # Run with FP32 datatype
+    sed -i 's/TYPE_INT32/TYPE_FP32/g' $MODEL_REPO/simple/config.pbtxt
+    sed -i 's/TYPE_INT32/TYPE_FP32/g' $MODEL_REPO/ensemble_add_sub_int32_int32_int32/config.pbtxt
+    $BASE_COMMAND -Dexec.args="-r $MODEL_REPO -v" >>$CLIENT_LOG 2>&1
+    if [ $? -ne 0 ]; then
+        echo -e "Failed to run: ${BASE_COMMAND} -Dexec.args=\"-r ${MODEL_REPO} -v\""
+        RET=1
```

Comment:
IIUC, it looks like you're editing the config.pbtxt file to replace INT32 -> FP32 for the [simple](https://github.com/triton-inference-server/server/tree/main/docs/examples/model_repository/simple) model, but the actual model file itself for simple expects INT32, so I'm not sure this will work. 

![image](https://user-images.githubusercontent.com/21284872/165831332-d79e985e-ce4f-441a-8971-99d24d8ad97e.png)